src/slurm_plugin/resume.py

```diff
@@ -157,18 +153,19 @@ def _handle_failed_nodes(node_list, reason="Failure when resuming nodes"):
     To save time, should explicitly set nodes to DOWN in ResumeProgram so clustermgtd can maintain failed nodes.
     Clustermgtd will be responsible for running full DOWN -> POWER_DOWN process.
     """
-    try:
-        log.info(
-            "Setting following failed nodes into DOWN state %s with reason: %s", print_with_count(node_list), reason
-        )
-        set_nodes_down(node_list, reason=reason)
-    except Exception as e:
-        log.error(
-            "Failed to place nodes %s into DOWN for reason %s with exception: %s",
-            print_with_count(node_list),
-            reason,
-            e,
-        )
+    if node_list:
```

Comment:
when we have reset the node failures with _reset_failed_nodes, it could happen that the error key is still there but there are no more nodes associated to that error